HW9.2/HW9.2.cpp

```diff
@@ -0,0 +1,25 @@
+﻿
+#include <iostream>
+
+void toUpperCase(char str[],int size)
+{
+	for (int i = 0; i < size; ++i) 
+	{
+		if (str[i] >= 'a' && str[i] <= 'z')
+		{
+			str[i] = str[i] - 'a' + 'A';
```

Comment:
calculate 'a' + 'A' once outside of the loop to not repeat the same calculation N times